Content.Server/DeviceNetwork/Systems/DeviceNetworkSystem.cs

```diff
@@ -62,10 +78,20 @@ public bool QueuePacket(EntityUid uid, string? address, NetworkPayload data, uin
             if (frequency == null)
                 return false;
 
-            _packets.Enqueue(new DeviceNetworkPacketEvent(device.DeviceNetId, address, frequency.Value, device.Address, uid, data));
+            _nextQueue.Enqueue(new DeviceNetworkPacketEvent(device.DeviceNetId, address, frequency.Value, device.Address, uid, data));
             return true;
         }
 
+        /// <summary>
+        /// Swaps the active queue.
+        /// Queues are swapped so that packets being sent in the current tick get processed in the next tick.
+        /// </summary>
```

Comment:
Maybe add clarification to avoid bugs in future refactors (i.e., "this prevents infinite recursion while sending packets")